src/th/niceoppai/src/eu/kanade/tachiyomi/extension/th/niceoppai/Niceoppai.kt

```diff
@@ -20,9 +22,9 @@ import java.util.Calendar
 import java.util.Locale
 import java.util.concurrent.TimeUnit
 
-class Niceoppai : ParsedHttpSource() {
+open class Niceoppai : ParsedHttpSource() {
```

Comment:
```suggestion
class Niceoppai : ParsedHttpSource() {
```

Don't need to be open as there's no inheritance from it.